ref/unitTests/interpolateTest.f90

```diff
@@ -0,0 +1,60 @@
+program interpolateTest
```

Comment:
I added a new directory under "ref" for the unit tests, called unitTests.  I choose to not put my unit test in "src", this in inline with Rob's unit tests (in his draft PR).

Reply:
Please remove that directory and put everything related to tests in a test/src directory.